Framework/Core/src/DataSamplingReadoutAdapter.cxx

```diff
@@ -0,0 +1,56 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "Framework/DataSamplingReadoutAdapter.h"
+#include "Headers/DataHeader.h"
+#include "Framework/DataProcessingHeader.h"
+
+namespace o2
+{
+namespace framework
+{
+
+using DataHeader = o2::header::DataHeader;
+
+InjectorFunction dataSamplingReadoutAdapter(OutputSpec const& spec)
+{
+
+  // copied from Common/DataBlock.h
```

Comment:
Why copying it?

Reply:
Maybe I was not specific enough - I meant Common repo in AliceO2Group, not Common inside O2.

Reply:
O2 does not depend on Common repo.

Reply:
Is this the actual readout header block? If yes, this looks like a good reason to make `Common` a dependency of `AliceO2` rather than copying it.

Reply:
Yes, it is the readout header block.
I would prefer `Common` to be `AliceO2` dependency as well, but I expected more resistance in that matter ;) Do you want to make it dependent yourself or should I do it?